tests/h2-support/src/future_ext.rs

```diff
@@ -72,20 +72,11 @@ pub trait FutureExt: Future {
     ///
     /// This allows the executor to poll other futures before trying this one
     /// again.
-    fn yield_once(self) -> Box<Future<Item = Self::Item, Error = Self::Error>>
+    fn yield_once(self) -> Box<dyn Future<Item = Self::Item, Error = Self::Error>>
```

Comment:
Do we care about supporting Rust versions that don't have the `dyn` keyword (such as 1.26.0?)

Reply:
No, it's too much of a pain. hyper already uses this, so onward!